lessons/Projects/project-video-game/langs/en-us/index.adoc

```diff
@@ -1,102 +1,145 @@
-* Share their completed video game with friends and family to play!
+This project can be used as the capstone for Bootstrap:Algebra. It is designed to provide real-world and engaging connections to the following mathematical concepts: Coordinates; Simple and Compound Inequalities; Domain and Range; Composing Functions; Connecting Representations and Defining Functions;
+Rotation, Dilation, and Translation; the Distance Formula and Pythagorean Theorem; Piecewise Functions.
 
-== Structure of the Game
 
-The code in the @starter-file{game} is divided into sections. Each section corresponds to a lesson with scaffolded activities and one or more workbook pages, which support them in writing a section of the game code. Students work in groups or pairs to customize and complete the code to make the game their own!
 
+=== Launch
 
-[cols=".^1a, .^1a, .^1a" options="header", stripes="none"]
+There are six phases of video game creation. Each phase (except for the first and the last) requires that you teach:
+
+- up to four lessons during which students learn relevant coding skills and algebra concepts
+- one lesson where students use what they've learned to update the code of their @starter-file{game, "Game Starter File"}
+
+In the first two phases (below), students conceptualize their game and then add images to their @starter-file{game, "Game Starter File"}. The table below highlights (1) the lessons you will teach to reinforce relevant algebra and coding concept, and (2) the lessons where students will update their personal video games.
+
+[cols="^.^1,<.^3,<.^6,<.^3", stripes="none", options="header"]
 |===
-| *Game Section*	| *Lessons* | *Workbook Pages*
-| *0{empty}.  Game title*
+| #
+| Phase
+| First, Learn Code and Algebra:
+| Then, Update the Game:
+
+| 1
+| Conceptualization
+| _NA_
 | @lesson-link{coordinates}
 
-| @printable-exercise{coordinates/pages/BrainstormGame.adoc}
+| 2
+| Game Imagery
+| @lesson-link{order-of-operations}, @lesson-link{simple-data-types}, @lesson-link{contracts}, @lesson-link{defining-values}
+| @lesson-link{making-game-images}
 
-| *1{empty}. Making the Danger and the Target Move*
-| @lesson-link{functions-for-character-animation}
+|===
+
+By the end of Phase 1, each student will have created a "screenshot" of their video game by inserting images that they find on the web into a @online-exercise{https://docs.google.com/drawings/d/1xXNWcbJ7KOgTClFQeHq-tIy0xmeZds9N3OktvEBaO4c/copy, Google Draw template}. Along the way, students will think about horizontal and vertical position as represented by ordered pairs.
+
+In Phase 2, students acquire the skillset needed to actually add game images to section 0 of their @starter-file{game, "Game Starter File"}. The programming concepts that students learn here form a foundation upon which to build, and will prove essential as you work your way through the remaining lessons. Students actually edit and update their games during @lesson-link{making-game-images}.
```

Comment:
Make a copy of the game templates that gets rid of section 0 and re-aligns them with the phases described here. No reason to have two separate lists of steps

Reply:
reulctant to do this because I'm not sure how to locate all other instances where we mention section numbers in a variety of lessons. Can you help out, maybe? There's probably something I could type into "Find in Files"?

Reply:
Hrmmm...I wish there was! I _think_ the game is the only place where we do this, however -- so in theory it should only be the lessons where kids work on the game.